components/putty/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Till Wegmueller
+#
+
+include ../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=putty
+COMPONENT_VERSION=0.70
+COMPONENT_SUMMARY=A terminal integrated SSH/Telnet client commandline tools
+COMPONENT_PROJECT_URL=http://www.chiark.greenend.org.uk/~sgtatham/putty/
+COMPONENT_FMRI=network/security/$(COMPONENT_NAME)
```

Comment:
We have no other fmris in network/security/* , perhaps, use just network/putty or security/putty?